blueflood-http/src/main/java/com/rackspacecloud/blueflood/inputs/handlers/HttpAggregatedIngestionHandler.java

```diff
@@ -99,10 +99,10 @@ public void handle(ChannelHandlerContext ctx, HttpRequest request) {
                         HttpResponseStatus.BAD_REQUEST );
             }
 
-        } catch (JsonParseException ex) {
+        } catch (JsonSyntaxException ex) {
```

Comment:
I think we should go back to catching ```JsonParseException``` because there are 2 exceptions subclassing ```JsonParseException```: one is ```JsonSyntaxException```, the other is ```JsonIOException```. We want to be able to catch both.